packages/klighd-core/src/depth-map.ts

```diff
@@ -393,6 +394,7 @@ export class DepthMap {
      * @returns Boolean value indicating the visibility of the region in the current viewport. 
      */
     isInBounds(region: Region, viewport: Viewport): boolean {
+        // TODO: only partially working, e.g. region "too late" out of bounds, see motor-copy
```

Comment:
this is also just unfortunate naming here, because depthmap regions are not the same as sccharts regions

Reply:
It was an old TODO, I removed it. Thanks for remark